cloud/blockstore/libs/storage/volume/partition_requests.h

```diff
@@ -57,7 +57,7 @@ bool ToPartitionRequests(
     Y_UNUSED(requests);
     Y_UNUSED(blockRange);
 
-    Y_ABORT_UNLESS(0);
+    ythrow TServiceError(E_NOT_IMPLEMENTED);
```

Comment:
`Y_ABORT_UNLESS(0, "Not implemented")`?

Reply:
nbs craches on Y_ABORT_UNLESS but we want to get readable error

so ythrow is better solution here